src/main/scala/sbttravisci/TravisCiPlugin.scala

```diff
@@ -14,7 +18,11 @@ object TravisCiPlugin extends AutoPlugin {
   override def trigger  = allRequirements
 
   override def globalSettings = Seq(
-    isTravisBuild := sys.env.get("TRAVIS").isDefined)
+    isTravisBuild := sys.env.get("TRAVIS").isDefined,
+    prBuildNumber := Try {
```

Comment:
Looks good. But I want to prefix the keys we introduce (it's a plugin best practice), so WDYT of `travisPrNumber`?